pkg/mirrorregistries/mirror_registry.go

```diff
@@ -0,0 +1,189 @@
+package mirrorregistries
```

Comment:
Maybe a better file name would be `parser.go`?
Seems a bit weird to just have this be the package name in a different format.